### PR TITLE
Improved CSS for JoinPage, Preference and fixed warnings on console for lobby for frontend

### PR DESCRIPTION
**Pull request by DavidHuangz**
Improved CSS for JoinPage, Preference and fixed warnings on console for lobby. Added more css files etc.

Addresses issue #[165](https://github.com/we-are-number-1/yumble/issues/165)

### CODE REVIEW

**Review by OverCry — DISMISSED** [review]
Looks good. Just not sure about 
image
but it should be fine since it is empty

**Comment by DavidHuangz** [comment]
> Looks good. Just not sure about
> image
> but it should be fine since it is empty

yea I deleted it. Fixed.

**Review by OverCry — DISMISSED** [review]
Looks good!

**Review by OverCry — DISMISSED** [review]
still looks good

**Review by ayushjani — APPROVED** [review]
LGTM

**Comment by DavidHuangz** [comment]
Rip, need another two quick review

**Review by ayushjani — APPROVED** [review]
Still looks good 👍

File: frontend/src/components/2.Join/JoinGroup.js [hunk 1]
```diff
@@ -33,33 +34,33 @@ function GroupCode() {
...
           <div className='GroupCode_Box'>
-            Enter group code
+            <div className='AddMargin'>Enter group code</div>
             <div>
               <UserInput
                 input
                 type='text'
                 inputType='joinGroup'
-                placeholder='e.g P6aPE'
+                placeholder=' P6aPE'
```
Comment (jemimahmartinez) [inline frontend/src/components/2.Join/JoinGroup.js:44]: Why did you take out the 'e.g.'? I feel like it might help the user?

Comment (DavidHuangz) [reply frontend/src/components/2.Join/JoinGroup.js:44]: I feel without the e.g looks better and also on many websites I've been to, they don't include that in the placeholder. But I can change it back.

**Review by OverCry — APPROVED** [review]
Still looks good

File: frontend/src/components/2.Join/JoinGroup.js (hunk 1 above)
Comment (jemimahmartinez) [reply frontend/src/components/2.Join/JoinGroup.js:44]: Aw nah I think thats alg, I took a look at it properly when I ran it and it looks better actually

**Review by jemimahmartinez — APPROVED** [review]
LGTM 👌

File: frontend/src/components/2.Join/JoinGroup.js (hunk 1 above)
Comment (DavidHuangz) [reply frontend/src/components/2.Join/JoinGroup.js:44]: yea, feel free to change it on your next commit if you feel like it might look better. :)

**Pull request merged**